packages/fastauth-metamask/README.md

```diff
@@ -0,0 +1,69 @@
+# @near-wallet-selector/fastauth-metamask
+
+This is the [fastauth-metamask](https://fastauth-metamask.app) package for NEAR Wallet Selector.
+
+## Installation and Usage
+
+The easiest way to use this package is to install it from the NPM registry, this package requires `near-api-js` v0.44.2 or above:
```

Comment:
When updating NAJ to latest we need to update the version here to `v1.0.0` or above.